Cargo.toml

```diff
@@ -25,6 +25,11 @@ rand = "0.8"
 levenshtein = "1.0"
 sqlx = { version = "0.6", features = ["runtime-tokio-rustls", "macros", "mysql", "bigdecimal", "chrono"]}
 base64 = "0.13"
+event_parser = { git = "https://github.com/allquixotic/event-parser", branch = "icalendar-bump" }
```

Comment:
I'll probably create a fork of this branch under this GitHub org, just to keep everything in one place

Reply:
Sounds good. The icalendar bump was the minimum work needed just to get this to compile, run, and return a reasonable result, but if you intend to (like you hinted in Discord) maintain event_parser longer term and add features, this is a great idea.

Reply:
Perhaps not fully maintain, but it certainly opens up some more possibilities if tweaks are required :)